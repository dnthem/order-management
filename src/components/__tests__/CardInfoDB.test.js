import CardInfoDB from "../CardInfoDB";
import renderer from 'react-test-renderer';

describe("CardInfoDB", () => {
    const props = {
        title: "title",
        value: "value",
        icon: "icon",
    }
        
    test("should render CardInfoDB component", () => {
        const component = renderer.create(
            <CardInfoDB {...props}/>,
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});