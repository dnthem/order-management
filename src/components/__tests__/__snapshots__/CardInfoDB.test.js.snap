// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardInfoDB should render CardInfoDB component 1`] = `
<div
  className="col-xl-3 col-md-6 mb-4 bg-"
>
  <div
    className="card border-left-primary shadow h-100 py-2"
  >
    <div
      className="card-body"
    >
      <div
        className="row no-gutters align-items-center"
      >
        <div
          className="col mr-2"
        >
          <div
            className="text-xs font-weight-bold text-primary text-uppercase mb-1"
          >
            title
          </div>
          <div
            className="h5 mb-0 font-weight-bold text-gray-800"
            data-test-id="card-info-value"
          >
            value
          </div>
        </div>
        <div
          className="col-auto"
        >
          icon
        </div>
      </div>
    </div>
  </div>
</div>
`;
